src/reconcile/rec_write.c

```diff
@@ -6591,3 +6594,37 @@ __rec_dictionary_lookup(
 	*dpp = next;
 	return (0);
 }
+
+/*
+ * __verbose_lookaside_write --
+ *	Create a verbose message to display with details about the transaction
+ * state when performing a lookaside table write.
+ */
+static void
+__verbose_lookaside_write(WT_SESSION_IMPL *session, uint64_t inserted_cnt)
+{
+#ifdef HAVE_VERBOSE
+	WT_SESSION *wt_session;
+	uint64_t pinned_range;
+	uint32_t pct_dirty, pct_full;
+
+	if (WT_VERBOSE_ISSET(session, WT_VERB_LOOKASIDE)) {
+
+		wt_session = &session->iface;
+
+		(void)__wt_eviction_clean_needed(session, &pct_full);
+		(void)__wt_eviction_dirty_needed(session, &pct_dirty);
+		wt_session->transaction_pinned_range(wt_session, &pinned_range);
```

Comment:
It's not normal for internal calls to redirect through a session handle. When that happens a number of fields get updated in the WT_SESSION structure. The general solution here would be to change ``__session_transaction_pinned_range`` to ``__wt__session_transaction_pinned_range`` and make it non-static.